src/redux/reducers/specPricesReducer.js

```diff
@@ -0,0 +1,21 @@
+import {
+  SET_SPEC_PRICES_SUCCESSFUL,
+  SET_SPEC_PRICES_FAILED,
+} from '../actionTypes';
+
+const initState = [];
```

Comment:
initState should be an object by default, the normalization process will return an object full of key values that use the id as a pointer